src/lang.js

```diff
@@ -367,6 +367,9 @@
 
             var leftDelimiter = matches[1];
             var leftNumber = convertNumber(matches[2]);
+            if (leftNumber === Infinity) {
```

Comment:
Got it @nanaya! As we are talking about the left number, if `converNumber` return a _positive_ `Infinity` it should because of the `*` or a mistake of the programmer.